app/views/api/clidocs/casestudy_metagenomics.html.erb

```diff
@@ -1,121 +1,434 @@
 <div class="apidocs casestudies row">
   <div class="col-xs-9">
-    <h1 class='api-title'>Case study: Metagenomics</h1>
-    <p class='lead'>
-      This case study describes how the Unipept command line tools can be used for the taxonomic analysis of metagenomics.
-    </p>
-    <div class="apidocs-callout apidocs-callout-danger" style="background-color: white">
-      <h4>Warning</h4>
-      <p>This is an (very incomplete) draft of how Unipept could be used for metagenomics. Proceed at your own risk.</p>
-    </div>
-    <div class='card'>
-      <div class='card-supporting-text'>
-        <p>
-          This is a short report on analysing the data from a shotgun metagenomics data set. For the moment, we have omitted the steps involved in quality filtering and mate pair assembly of the raw DNA reads, as well as gene prediction using <code>FragGeneScan</code>. The end result of all these steps are two files:
-    </p>
-        <ul>
-          <li><code>joined_proteins.fasta.faa</code>: predicted protein fragments on assembled reads that passed the quality filtering.</li>
-          <li><code>single_proteins.fasta.faa</code>: predicted protein fragments on non-assembled reads that passed the quality filtering.</li>
-        </ul>
-        <p>
-          In a first step, we process all protein fragments (both from the assembled and single reads) by splitting them into tryptic peptides (filtered to length in the range [5, 50]) and merging the peptides coming from the same DNA read into a single FASTA record <i>(timing: 12 mins 22 secs)</i>.
-    </p>
-        <pre>
-<b>$</b> cat joined_proteins.fasta.faa singles_proteins.fasta.faa | prot2pept | peptfilter | python3 join_reads.py > peptides.fst</pre>
+    <h1 class="api-title">Case Study: Unipept Metagenomics Analysis Pipeline</h1>
+    <p class="lead">This case study show how to use UMGAP for the taxonomic analysis of a metagenomics dataset.</p>
```

Comment:
`shows`

_[Original comment](https://github.ugent.be/unipept/unipept/pull/642#discussion_r16543) by @bmesuere on Mon Aug 20 2018 at 13:36._